src/Microsoft.ML.Core/Utilities/DoubleParser.cs

```diff
@@ -12,26 +12,26 @@ namespace Microsoft.ML.Internal.Utilities
     [BestFriend]
     internal static class DoubleParser
     {
+        [BestFriend]
+        [Flags]
+        internal enum OptionFlags : uint
+        {
+            Default = 0x00,
+
+            // If this flag is set, then a "," will be used as Decimal Marker
+            // (i.e., the punctuation mark that separates the integer part of
+            // a number and its decimal part). If this isn't set, then
+            // default behavior is to use "." as decimal marker.
+            UseCommaAsDecimalMarker = 0x01,
+
+            All = UseCommaAsDecimalMarker
+        }
```

Comment:
I don't think we should have the All option here. As we add more items to OptionFlags later, the meaning of All will keep changing and might break code that uses All. It is better for each caller to explicitly make use of only what they need.

Reply:
You're right. Fixed.

---
In reply to: [429586349](https://github.com/dotnet/machinelearning/pull/5154#discussion_r429586349) [](ancestors = 429586349)